ROOTFS/etc/waggle/sanity/fatal/core_sdcard.test

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+pf="Recovery SDCard test"
+echo "$pf: Begin"
+
+# run fsck on our sd card
+if [ "$(fsck -n /dev/mmcblk1p1 | grep clean)" = "" ];
+then
+        echo "$pf: /dev/mmcblk1p1 failed fsck FAIL"
+        exit 1
+fi
+
+if [ "$(fsck -n  /dev/mmcblk1p2 | grep clean)" = "" ];
+then
+        echo "$pf: /dev/mmcblk1p2 failed fsck FAIL"
+        exit 2
+fi
+
+mkdir /tmp/core_sdcard_test
+
+mount /dev/mmcblk1p1  /tmp/core_sdcard_test/   
```

Comment:
If we are in recovery mode these tests will not be able to run because the SD card will already be mounted. Since this file is meant for running all tests relating to the SD card maybe we could check whether we are in recovery mode. If not, the tests we are currently running look good. If we are, we should run similar tests to what we check for on the NVME

```
# properly mounted tests 
 if [ "$(df -h | grep "/media/root-ro")" = "" ];
 then
         echo "$pf: RO partition not mounted FAIL"
         exit 1
 fi

 if [ "$(df -h | grep "/media/root-rw")" = "" ];
 then
         echo "$pf: RW partition not mounted FAIL"
         exit 2
 fi

 if [ "$(df -h | grep "/media/plugin-data")" = "" ];
 then
         echo "$pf: plugin-data not mounted FAIL"
         exit 3
 fi
# checking initrd test
 if [ "$(file -L /boot/initrd | grep "gzip compressed data")" = "" ];
 then 
         echo "$pf: initrd is not a valid archive FAIL"
         exit 4
 fi
```

Reply:
Please ignore, tests will not be run in recovery mode

Reply:
pls mount read-only